packages/backend/src/extension.ts

```diff
@@ -26,9 +26,69 @@ import { HistoryNotifier } from './history/historyNotifier';
 import { Messages } from '/@shared/src/messages/Messages';
 import { satisfies, minVersion, coerce } from 'semver';
 import { engines } from '../package.json';
+import * as macadamJSPackage from '@crc-org/macadam.js';
+import { isHyperVEnabled, isWSLEnabled } from './macadam/win/utils';
+import { getErrorMessage, verifyContainerProivder } from './macadam/utils';
+import { LoggerDelegator } from './macadam/logger';
+import { ProviderConnectionShellAccessImpl } from './macadam/macadam-machine-stream';
+import { macadamName } from './constants';
 
 export const telemetryLogger = extensionApi.env.createTelemetryLogger();
 
+const EXTENSION_DESCRIPTION_MARKDOWN = `Bootable container-based virtual machines run via Macadam`;
+
+let stopLoop = false;
+
+type StatusHandler = (name: string, event: extensionApi.ProviderConnectionStatus) => void;
+const macadamMachinesInfo = new Map<string, MachineInfo>();
+const currentConnections = new Map<string, extensionApi.Disposable>();
```

Comment:
Unsure what the above do / no comments. I'm assuming that these are copypasta from the rhel vm extension?